client/my-sites/activity/filterbar/date-range-selector.jsx

```diff
@@ -47,7 +47,13 @@ export class DateRangeSelector extends Component {
 		} );
 
 		const formattedFromDate = fromDate && moment( fromDate ).format( DATE_FORMAT );
-		const formattedToDate = toDate && moment( toDate ).format( DATE_FORMAT );
+		const formattedToDate =
+			toDate &&
+			moment( toDate )
```

Comment:
could it be `toDate + 24 hours` instead?

Reply:
moment.endOf( 'day' )  FTW!!!

Reply:
:magnifique: